packages/eslint-plugin/tests/rules/prefer-nullish-coalescing.test.ts

```diff
@@ -1145,42 +1241,6 @@ x || y;
     {
       code: `
 declare const x: 0 | 1 | 0n | 1n | undefined;
-x || y;
-      `,
-      output: null,
-      options: [
-        {
-          ignorePrimitives: {
-            string: true,
-            number: false,
-            boolean: true,
-            bigint: true,
-          },
-        },
-      ],
-      errors: [{ messageId: 'preferNullishOverOr' }],
-    },
```

Comment:
I believe these tests to have been wrong originally; this says "ignore bigint", and there's a bigint in the union, but the union is not ignored. Now moved up to "valid" (with the other similar tests).